pom.xml

```diff
@@ -3,12 +3,12 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.5</version>
+    <version>3.37</version>
   </parent>
   
   <properties>
-    <jenkins.version>2.7.4</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.89.4</jenkins.version>
```

Comment:
why only 2.89.4?
Properly better to go with: 2.107.3 to avoid using `Jenkins.getInstanceOrNull` and just use `Jenkins.get`

Reply:
http://stats.jenkins.io/pluginversions/vstestrunner.html

perhaps I'd suggest just sticking with 2.60.3 just for Java 8 there aren't that many changes if you remove the use of `Jenkins.getInstanceOrNull` and stick to `Jenkins.getInstance`

Reply:
I simply looked at what LTS was released a year ago. I'm fine with newer. I'll update the PR.